src/sentry/static/sentry/app/views/organizationIncidents/list/index.jsx

```diff
@@ -63,4 +63,21 @@ class OrganizationIncidents extends AsyncView {
   }
 }
 
+class OrganizationIncidents extends React.Component {
+  render() {
+    return (
+      <DocumentTitle title={`Incidents - ${this.props.params.orgId} - Sentry`}>
+        <PageContent>
```

Comment:
I feel like we use this pattern enough where we should extend `AsyncView` (whether it's in that component or a new one) to include this.